core/domain/stats_jobs_one_off.py

```diff
@@ -110,10 +110,19 @@ def map(item):
             })
 
         elif isinstance(item, stats_models.StateHitEventLogEntryModel):
+            try:
```

Comment:
Why don't you just do `if state_name is None`?

Reply:
Done.